carma_wm_ctrl/include/carma_wm_ctrl/WMBroadcaster.h

```diff
@@ -381,6 +383,24 @@ class WMBroadcaster
   /*! \brief helps to populate upcoming_intersection_ids_ from local traffic lanelet ids
    */
   void publishLightId();
+  
+  /*!
+   * \brief Retrieve the vehicle ID from global vehicle parameters, and set instance memeber vehicle id
+   @param vehicle_id Vehicle ID from UniqueVehicleParams.yaml 
+   */
+  void setConfigVehicleId(std::string vehicle_id);
```

Comment:
string should be passed as const reference to prevent unneeded copy
const std::string& vehicle_id